<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Because Routing</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="serviceLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Routing Service</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="serviceChoice">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="maxCount">
      <number>20</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <item>
      <property name="text">
       <string>MapBox</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MapZen</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="originAddressLabel">
     <property name="text">
      <string>Origin Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="originAddress">
     <property name="placeholderText">
      <string>1600 Pennsylvania Ave, Washington, DC</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="destinationAddressLabel">
     <property name="text">
      <string>Destination Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="destinationAddress">
     <property name="placeholderText">
      <string>1000 Jefferson Dr SW, Washington, DC 20560</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="layerNameLabel">
     <property name="text">
      <string>Layer Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="layerName">
     <property name="toolTip">
      <string>Choose a name for the layers to be created.</string>
     </property>
     <property name="placeholderText">
      <string>My Route</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacerBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>serviceChoice</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
